assets/queries/cloudFormation/api_gateway_xray_disabled/query.rego

```diff
@@ -0,0 +1,39 @@
+package Cx
+
+CxPolicy[result] {
+	document := input.document
+	resource = document[i].Resources[name]
+	resource.Type == "AWS::ApiGateway::Stage"
+	properties := resource.Properties
+	isResFalse(properties.TracingEnabled)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties.TracingEnabled", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("Resources.%s.Properties.TracingEnabled is false", [name]),
+		"keyActualValue": sprintf("Resources.%s.Properties.TracingEnabled is true", [name]),
```

Comment:
Swapped keyExpected and keyActual? Should be:

"keyExpectedValue": sprintf("Resources.%s.Properties.TracingEnabled is true", [name]),
		"keyActualValue": sprintf("Resources.%s.Properties.TracingEnabled is false", [name]),